Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,11 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
```

Comment:
What is a `shortname`? that's not a term we use elsewhere.

Reply:
Shortname was originally used in homebrew's bash completion:

```bash
# completions/bash/brew:53 

__brew_complete_formulae() {
  local cur="${COMP_WORDS[COMP_CWORD]}"
  local formulas="$(brew search)"
  local shortnames="$(echo "$formulas" | \grep / | \cut -d / -f 3)"
  COMPREPLY=($(compgen -W "$formulas $shortnames" -- "$cur"))
}
```

For completion, we need both the full formula path ( `repo_owner/tap_name/formula_name`) as well as simply `formula_name`.  `shortnames` is just a temporary holder for the formula names from taps without the rest of the path.  That does mean there will be duplicated names, though every line is still unique.  

Depending on what you think the behavior should be, we could either:

1.  Leave it as is
2.  Add an argument (`--include-shortened-names` or sometihng similar.  I am not sure what the brew terminology is or if it even exists) that will include those extra names for completions, but by default there will be only the full names
3. Omit it entirely and move this logic into the completions themselves.  
4. Do something else entirely

I have no preference personally, and I don't think it would impact performance all that much.

Reply:
```suggestion
#:  Prints a sorted list of locally available formulae.
```

I would just go for this instead, I think it's fine 👍🏻 

Are you saying this should include both:
```
wget
some/tap/wget
``` 

in the output. If so could be:

```suggestion
#:  Prints a sorted list of available formulas names with and without tap prefixes.
```